pandas/window.pyx

```diff
@@ -1285,6 +1285,9 @@ def roll_quantile(ndarray[float64_t, cast=True] input, int64_t win,
         ndarray[int64_t] start, end
         ndarray[double_t] output
 
+    if quantile < 0.0 or quantile > 1.0:
```

Comment:
Can use a single `0 <= quantile <= 1` statement. Mind the endpoints: can't the chosen quantile be 0 or 1?